smartmin/users/views.py

```diff
@@ -316,6 +316,11 @@ def form_valid(self, form):
                 send_mail(_('Password Recovery Request'), email_template.render(context), from_email,
                             [email], fail_silently=False)
 
+            # Run the hook function taking parameters user, email
+            user_forget_hook_function = getattr(settings, "USER_FORGET_HOOK_FUNCTION", None)
```

Comment:
this feels like a job for a custom Django signal rather that a function in settings... @nicpottier ?